### DISCUSSION TITLE
Integration with IBM carbon components (MultiSelect)

### DISCUSSION THREAD

**Discussion by dankobg** [pablo-abc/felte#167] [Q&A]
Hi, does anyone use this library with [carbon-components-svelte](https://carbon-components-svelte.onrender.com/)?
I am trying to integrate with it and it works fine for normal fields but i don't know how to make it work nicely with MultiSelect component for example or ComboBox.
If i bind value to the array of selected IDs for the MultiSelect, it gets weird and it shows those values inside an input.
So i figured i might as well ask here, somebody surely had to use this component library.
Any guidance, repl or codesandbox will be helpful.

ps: this is roughly how it looks like
[repl example](https://svelte.dev/repl/4c893fa751ae4601bf6c5bf619550f72?version=3.49.0)
adding display: none on the input with role="combobox" works but the checkboxes are not updating immediately after selecting..

**Comment by rbaprado** [comment]
> Hi, does anyone use this library with [carbon-components-svelte](https://carbon-components-svelte.onrender.com/)? I am trying to integrate with it and it works fine for normal fields but i don't know how to make it work nicely with MultiSelect component for example or ComboBox. If i bind value to the array of selected IDs for the MultiSelect, it gets weird and it shows those values inside an input. So i figured i might as well ask here, somebody surely had to use this component library. Any guidance, repl or codesandbox will be helpful.
> 
> ps: this is roughly how it looks like [repl example](https://svelte.dev/repl/4c893fa751ae4601bf6c5bf619550f72?version=3.49.0) adding display: none on the input with role="combobox" works but the checkboxes are not updating immediately after selecting..

@dankobg How did you work around Carbon's Form component not allowing the "use:form" directive?
